src/lib/import-export/tests/export/export-manager.test.ts

```diff
@@ -1,96 +1,63 @@
-import fs from 'fs/promises';
```

Comment:
what would happen in case no exporter can handle the export? should we cover this in a test case?

Reply:
I think this was missing from the original implementation. As of now, nothing will happen but i assume that once the progress bar is implemented, this will be addressed.